runtime/include/tasks/qthreads/tasks-qthreads.h

```diff
@@ -117,6 +117,16 @@ extern
 volatile int chpl_qthread_done_initializing;
 
 typedef struct {
+  //chpl_bool serial_state;
```

Comment:
Remove commented-out members.  But, it seems worthwhile to move the lineno and filename into the common code.  All the layer implementations need these two, after all.